war/src/main/webapp/scripts/behavior.js

```diff
@@ -95,21 +95,25 @@ var Behaviour = (function() {
                 Behaviour.list._each(function(sheet) {
                     for (var selector in sheet){
                         startNode._each(function (n) {
+                          try {
                             var list = findElementsBySelector(n, selector, includeSelf);
                             if (list.length > 0) { // just to simplify setting of a breakpoint.
                                 //console.log('deprecated:' + selector + ' on ' + list.length + ' elements');
                                 list._each(sheet[selector]);
                             }
+                          } catch (e) {}
```

Comment:
Not even logging to console? Is this really not a relevant error?

Same question applies to below.

Reply:
@jsoref ?

Reply:
You can add logging... It was a long time ago, you might drown. Try locally...

Reply:
It's currently trying 2 different methods, and here's the failures if you add logging:

![image](https://user-images.githubusercontent.com/21194782/81466877-caccd980-91cc-11ea-9fbe-a861dbdc78d0.png)

Reply:
No logging then 😄

Reply:
Well, some of those at least are actually legitimate errors (Global properties aren't being hidden if checkbox is unchecked), and at least at one point in this PR both myself and Felix hit errors being swallowed and looks like this was the cause, it needs fixing

Reply:
this line is only getting invoked, ~6 times, although if I query the selector it matches approx ~14 elements.

https://github.com/jsoref/jenkins/blob/2c2d08f2d7e0622d02ff9e6f46399b18ba05bed8/war/src/main/webapp/scripts/hudson-behavior.js#L1070

Reply:
Looks like the issue is here:
https://github.com/jenkinsci/matrix-auth-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/matrixauth/AuthorizationMatrixNodeProperty/config.groovy#L10

the table needs to be a div otherwise the `rowvg-end` is in the wrong place

Reply:
I get no errors when I disable 'Project-based Matrix Authorization Strategy'

I'll send a PR to matrix-auth which makes it compatible with both versions

Reply:
so um does anyone know how to access a jelly property in a groovy view 🤦 ?

@daniel-beck ?

(trying to adapt based on divBasedFormLayout)

Reply:
Isn't this the same case as Artifactory Plugin, solved by a new wrapper tag?

FWIW I would check in `context` which is the `JellyContext`.

Reply:
`getProperty` is a global property that allows this, no reference to it in Jenkins core, found it by stepping through the debugger and browsing the code

Reply:
https://github.com/jenkinsci/matrix-auth-plugin/pull/83